### PR TITLE
Upgraded the nokogiri dependency and update the version number

### PR DESCRIPTION
**Pull request by mvsBR**
Upgraded the nokogiri dependency which is required for it's use in rails 4 applications.

Here are the logs of the test after updating the version.

```
W, [2024-07-03T12:45:51.128657 #108512]  WARN -- : You are setting a key that conflicts with a built-in method OmniAuth::AuthHash::InfoHash#name defined at /home/bari/.rvm/gems/ruby-2.5.7/gems/omniauth-1.2.2/lib/omniauth/auth_hash.rb:34. This can cause unexpected behavior when accessing the key as a property. You can still access the key via the #[] method.
..............................I, [2024-07-03T12:45:51.481606 #108512]  INFO -- omniauth: (wsfed) Request phase initiated.
.I, [2024-07-03T12:45:51.484902 #108512]  INFO -- omniauth: (wsfed) Request phase initiated.
.I, [2024-07-03T12:45:51.485741 #108512]  INFO -- omniauth: (wsfed) Request phase initiated.
.I, [2024-07-03T12:45:51.486254 #108512]  INFO -- omniauth: (wsfed) Request phase initiated.
.I, [2024-07-03T12:45:51.487189 #108512]  INFO -- omniauth: (wsfed) Callback phase initiated.
E, [2024-07-03T12:45:51.489803 #108512] ERROR -- omniauth: (wsfed) Authentication failure! invalid_authn_token: OmniAuth::Strategies::WSFed::ValidationError, AuthN token (wresult) missing in callback.
.

Deprecation Warnings:

Using `should` from rspec-expectations' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` with `config.expect_with(:rspec) { |c| c.syntax = :should }` instead. Called from /home/bari/Development/meazure-learning/code/omniauth-wsfed/spec/omniauth/strategies/wsfed/auth_callback_spec.rb:32:in `block (4 levels) in <top (required)>'.

Using `stub` from rspec-mocks' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` instead. Called from /home/bari/Development/meazure-learning/code/omniauth-wsfed/spec/omniauth/strategies/wsfed/auth_callback_validator_spec.rb:20:in `block (2 levels) in <top (required)>'.


If you need more of the backtrace for any of these deprecations to
identify where to make the necessary changes, you can configure
`config.raise_errors_for_deprecations!`, and it will turn the
deprecation warnings into errors, giving you the full backtrace.

2 deprecation warnings total

Finished in 0.2458 seconds (files took 0.26415 seconds to load)
35 examples, 0 failures
```

### CODE REVIEW

**Comment by msharma-ml** [comment]
@mvsBR Could you please share here the logs after running test with new version of the gem.

File: lib/omniauth-wsfed/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module WSFed
-    VERSION = '0.2.3'
```
Comment (msharma-ml) [inline lib/omniauth-wsfed/version.rb:3]: Do we need to change the patch version here @mvsBR ?

Or we can release a new tag for the gem `0.2.3.2`.

cc: @ad-ys

**Review by msharma-ml — CHANGES_REQUESTED** [review]

File: lib/omniauth-wsfed/version.rb (hunk 1 above)
Comment (ad-ys) [reply lib/omniauth-wsfed/version.rb:3]: yes lets bump this to `0.2.3.2` and tag it accordingly

also i think we are actually using this branch https://github.com/yardstick/omniauth-wsfed/commits/yardstick-ruby-2.5/

Comment (mvsBR) [reply lib/omniauth-wsfed/version.rb:3]: Updated the version number and created a new PR taking base of `yardstick-ruby-2.5` branch.

Closing this one.

**Pull request closed without merging**